js/huobi.js

```diff
@@ -5216,9 +5216,12 @@ module.exports = class huobi extends Exchange {
             //     }
             //
         }
-        const request = {
-            'contract_code': market['id'],
-        };
+        const request = {};
+        if (marketType === 'future') {
+            request['symbol'] = market['baseId'];
```

Comment:
In futures, we should only send the base currency as "symbol"
![image](https://user-images.githubusercontent.com/43336371/154997409-2cbae1f4-d77a-43da-bded-aa941f8fe815.png)

Reply:
See how we do it here: 

- https://github.com/ccxt/ccxt/blob/master/js/huobi.js#L2581
- https://github.com/ccxt/ccxt/blob/master/js/huobi.js#L2585
- https://github.com/ccxt/ccxt/blob/master/js/huobi.js#L2597

Reply:
@kroitor just changed from '`market['baseId'] -> market['settleId], `, is that it right?

Reply:
Nope, I mean, check those three lines linked above to see how we decide which parameter to send `contract_code` and/or `symbol` depending on the market type.

Reply:
The baseId is not the same as settleId, for sure.

Reply:
@kroitor since fetchPositions does not handle spot markets, isn't what I'm doing equivalent to that?
If it's a inverse future use "symbol", otherwise "contract_code"
![image](https://user-images.githubusercontent.com/43336371/155107475-23575e7b-cd11-41b0-8428-37cf0bebb21a.png)

Reply:
@carlosmiei yep, it's ok, I justed added that for clarity.